src/aws-wa-sec-tool-mcp-server/awslabs/aws_wa_sec_tool_mcp_server/server.py

```diff
@@ -0,0 +1,1196 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""AWS Well-Architected Security Assessment Tool MCP Server"""
+
+import argparse
+import asyncio
+import datetime
+import os
+import sys
+from typing import Dict, List, Optional
+
+import boto3
+from botocore.config import Config
+from loguru import logger
+from mcp.server.fastmcp import Context, FastMCP
+from pydantic import Field
+
+from awslabs.aws_wa_sec_tool_mcp_server.consts import INSTRUCTIONS
+from awslabs.aws_wa_sec_tool_mcp_server.util.network_security import check_network_security
+from awslabs.aws_wa_sec_tool_mcp_server.util.resource_utils import list_services_in_region
+from awslabs.aws_wa_sec_tool_mcp_server.util.security_services import (
+    check_access_analyzer,
+    check_guard_duty,
+    check_inspector,
+    check_macie,
+    check_security_hub,
+    check_trusted_advisor,
+    get_access_analyzer_findings,
+    get_guardduty_findings,
+    get_inspector_findings,
+    get_macie_findings,
+    get_securityhub_findings,
+    get_trusted_advisor_findings,
+)
+from awslabs.aws_wa_sec_tool_mcp_server.util.storage_security import check_storage_encryption
+
+# User agent configuration for AWS API calls
+USER_AGENT_CONFIG = Config(user_agent_extra="awslabs/mcp/aws-wa-sec-tool-mcp-server/1.0.0")
```

Comment:
would need to add `rom awslabs.aws_wa_sec_tool_mcp_server import __version__`
```suggestion
USER_AGENT_CONFIG = Config(user_agent_extra="awslabs/mcp/aws-wa-sec-tool-mcp-server/{__version__}")
```